j/jamm/jamm-master-ubi8.4.sh

```diff
@@ -0,0 +1,37 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: jamm
+# Version	: master
```

Comment:
Any specific reason for building master ?
Because, master branch will keep on changing and we can't say master branch build is stable.

Reply:
Requested version was 0.3.0 which is from 2014. This version has issue and need fix in maven url in build.xml. Hence as suggested by Priya created two scripts one for master without any patch and another for requested version with patch.

Reply:
Okay. Thanks for the details.

In that case, can you check out the current commit id on master branch (instead of using master, use a latest commit id) as version ? That makes our build script stable, even if anything added/failed on master in future, it won't effect our build script.

Reply:
I have tried with latest stable version 0.3.3. But this version also has same issue. Only master code is building without patch.

Reply:
changes committed